torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1842,6 +1842,9 @@ def sample_inputs_norm(op_info, device, dtype, requires_grad, **kwargs):
         ((S, S), (-2,), 'neg_2'),
         ((S, S), (-0.5,), 'neg_0_5'),
         ((S, S), (-1.5,), 'neg_1_5'),
+        # FIXME gradgrad and noncotiguous_samples fails for these on CPU, how about that
+        # ((S, S), (inf,), 'inf'),
+        # ((S, S), (-inf,), 'neg_inf'),
```

Comment:
This was an interesting find, and it seems to be related to the odd fix `linalg.vector_norm` has. See https://github.com/pytorch/pytorch/pull/76547#discussion_r861201122.

Reply:
You can conditionally include these when CUDA samples are requested